### ISSUE TITLE
wireguard

### ISSUE THREAD

**Issue by rvpn** [Jigsaw-Code/outline-server#411]
hi sir, is it possible to add a support for wireguard too?

**Comment by shaomeng** [comment]
I was looking into wireguard the other day. It seems to me the major benefit of it is its lightweight nature, and the major downside is that it isn't considered mature yet. Is this also what you found out?

**Comment by rvpn** [comment]
> I was looking into wireguard the other day. It seems to me the major benefit of it is its lightweight nature, and the major downside is that it isn't considered mature yet. Is this also what you found out?

yes it's true. but this article give thorough explanation about wireguard 

https://restoreprivacy.com/wireguard/
